source/loader/layers/sanitizer/msan/msan_options.cpp

```diff
@@ -0,0 +1,90 @@
+/*
+ *
+ * Copyright (C) 2024 Intel Corporation
+ *
+ * Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+ * See LICENSE.TXT
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ * @file msan_options.cpp
+ *
+ */
+
+#include "msan_options.hpp"
+
+#include "ur/ur.hpp"
+#include "ur_sanitizer_layer.hpp"
+
+#include <algorithm>
+#include <cstring>
+#include <stdexcept>
+
+namespace ur_sanitizer_layer {
+namespace msan {
+
+MsanOptions::MsanOptions() {
```

Comment:
This is a near 1:1 copy/paste from asan_options.

Reply:
We consider to use a single sanitizer option in the future